src/contexts/dices.jsx

```diff
@@ -38,6 +39,15 @@ const DicesProvider = ({ children }) => {
     return dice
   }
 
+  if (!init) {
+    setInit(true)
+
+    if (
+      !dices
+      || Object.values(dices).length === 0
+    ) add({ faces: 6 })
```

Comment:
maybe you should update the default value for faces in `add` function
https://github.com/fabienjuif/roll/pull/13/files#diff-ca0d8dea7f1b1ca2221f92d55acf68feR26

Reply:
I just updated it, but I keep this line like that, this more understandable